migrations/db/migrations/20250421084701_revoke_supabase_storage_admin_from_postgres.sql

```diff
@@ -0,0 +1,10 @@
+-- migrate:up
+revoke supabase_storage_admin from postgres;
+revoke create on schema storage from postgres;
+revoke all on storage.migrations from anon, authenticated, service_role, postgres;
+
+revoke supabase_auth_admin from postgres;
+revoke create on schema auth from postgres;
+revoke all on auth.schema_migrations from dashboard_user, postgres;
```

Comment:
The `revoke all` privileges on the migrations tables are not covered by tests. We should add those to ensure no regressions.